pkgs/development/libraries/gstreamer/qt-gstreamer/default.nix

```diff
@@ -28,5 +28,6 @@ stdenv.mkDerivation rec {
   meta = with stdenv.lib; {
     platforms = platforms.linux;
     license = licenses.lgpl21;
+    broken = true;
```

Comment:
another distro already retired this https://src.fedoraproject.org/rpms/qt-gstreamer/c/92c2d302cbb8637430bdff3662599600113c0e8c?branch=master :+1: